src/I18n/Number.php

```diff
@@ -129,6 +130,24 @@ public static function format($value, array $options = [])
     }
 
     /**
+     * Parse a localized numeric string and transform it
+     * in a float point
+     *
+     * @param string $value A numeric string.
+     * @param string|null $locale Locale of the formatted number
+     * @return float point number
+     */
+    public static function parseFloat($value, $locale = null)
+    {
+        if ($locale === null) {
+            $locale = Locale::getDefault();
+        }
+
+        $formatter = new NumberFormatter($locale, NumberFormatter::DECIMAL);
+        return (float)$formatter->parse($value, NumberFormatter::TYPE_DOUBLE);
+    }
+
+    /**
```

Comment:
I follow the same logic of Time and DateTimeType.

One lib supporting this can be useful, for me at least - working with user data at Shell or Api/Controller, for example.

But yes, if you think this should restrict at Database layer, I can move the method.